src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
+spring.datasource.url=jdbc:mysql://localhost/mrdb_schema
+spring.datasource.username=root
+spring.datasource.password=Gurutivi@2105
```

Comment:
We should never commit passwords to github. When deployed live in production, these would come in as environment variables. For your local development, you can create another `application-local.properties` file with passwords, and configure your spring profile as “local” to pick the configuration from this file. You can then add this file to gitignore.

Reply:
Will make these changes in the next sprint.